lib/Main.js

```diff
@@ -50,6 +50,8 @@ function Main(config) {
 
     this._gitterRateLimiter = new RateLimiter(GITTER_LIMITER_INTERVAL);
 
+    this._gitterJoinedRooms = [];
```

Comment:
This looks unused

Reply:
Oh you removed this entire commit in the next one. I can only highly recommend `git commit --amend` locally before you push anything.